src/components/Lobby/LobbyChatList.tsx

```diff
@@ -19,36 +18,29 @@ interface Props {
   onLoadMore: Function;
   canLoadMore: boolean;
   onSendMessage: Function;
-  onToggleLobby: Function;
-  showFullLobby: boolean;
+  lastAddedItemId: string;
 }
 
-const LobbyChatList = ({
+const LobbyChatListMobile = ({
```

Comment:
bro. This file is "LobbyChatList.tsx" while "LobbyChatListMobile" we already have in its own file "LobbyChatListMobile.tsx" so please revert the component variable name here.

Reply:
solved here https://github.com/goffxnca/yallyo-web/pull/5/commits/7a488e95101c5484336d488f800c9b850e60a9db